pkg/machine/provider/platform_darwin.go

```diff
@@ -1,4 +1,4 @@
-//go:build darwin && arm64
+//go:build (darwin && arm64) || (darwin && amd64)
```

Comment:
why not just `darwin`, there are no other arches for it?
and given the file is already suffixed `_darwin` you can just remove the tag at all.